build.gradle

```diff
@@ -10,7 +10,7 @@ apply plugin: 'maven-publish'
 import org.apache.tools.ant.filters.*
 
 group 'io.appium'
-version '6.1.0'
+version '6.1.1-SNAPSHOT'
```

Comment:
don't we call it BETA1?

Reply:
Ideally we used to call as 'SNAPSHOT' in maven world before we release either beta or major version.